projects/js-packages/connection/README.md

```diff
@@ -0,0 +1,41 @@
+Connection Package
+=========
+
+The component encapsulates the Connection functionality.
+
+## Component InPlaceConnection
+It includes:
+- the `iframe` HTML element
+- connection URL handling
+- catching the "close" message and executing the callback
+- fallback for not available cookie
+
+### Properties
+- *title* - string (required), the iframe title.
+- *isLoading* - boolean, whether to display the "Loading..." label in the component, defaults to `false`.
+- *width* - string|number, the iframe width, defaults to `100%`.
+- *height* - string|number, the iframe height, defaults to `220`.
+- *scrollToIframe* - boolean, whether after iframe rendering, window should scroll to its current position. Defaults to `false`.
+- *onComplete* - callback, to be executed after connection process has completed.
+- *onThirdPartyCookiesBlocked* - callback, to be executed if third-party cookies are blocked.
+- *connectUrl* - string (required), the connection URL.
+- *displayTOS* - boolean (required), whether the iframe should display TOS or not.
+- *source* - string, component location identifier passed to WP.com.
+
+### Usage
+```jsx
+import InPlaceConnection from 'in-place-connection';
+
+<InPlaceConnection
+	connectUrl={ this.props.connectUrl }
+	height={ this.props.height }
+	width={ this.props.width}
+	isLoading={ this.props.fetchingConnectUrl }
+	title={ this.props.title }
+	hasConnectedOwner={ this.props.hasConnectedOwner }
```

Comment:
```suggestion
	displayTOS={ false }
```

Also (non trivial) we could maybe replace `this.props` logic here with actual example values to make it easier to understand.

Finally (again non trivial) we could also mention the nice fallback helper for 3rd party cookies you added!

Reply:
You're right, we need the docs!
Done!